### PR TITLE
Logs Check Contents usage

### PR DESCRIPTION
**Pull request by Ivniinvi**
## About The Pull Request
Usage of Check Contents is now logged, and the log shows what the contents were at the time of the check.

## Why It's Good For The Game
Far, far easier to process powergaming appeals now.

## Changelog
:cl:
admin: Contents are now logged when a Check Contents occurs
/:cl:

### CODE REVIEW

File: code/modules/admin/verbs/randomverbs.dm [hunk 1]
```diff
@@ -712,6 +712,7 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	var/list/L = M.get_contents()
 	for(var/t in L)
 		to_chat(usr, "[t]")
+	log_game("[key_name(usr)] checked the contents of [key_name(M)], which were [M.get_contents()]")
```
Comment (ike709) [inline code/modules/admin/verbs/randomverbs.dm:715]:
```suggestion
	log_admin("[key_name(usr)] checked the contents of [key_name(M)], which were [M.get_contents()]")
```

**Comment by ike709** [comment]
I'm not sure how I feel about dumping the entire contents into it. Also do you really need to call the proc a second time? You should just `jointext()` the list. Actually while you're here you could remove the for loop, call `jointext()` to make a single string, and output that to both the usr and the log.

EDIT: To clarify what I'm saying, just use `jointext()` on `L` instead of doing the second `get_contents()` call.

File: code/modules/admin/verbs/randomverbs.dm (hunk 1 above)
Comment (Archanial) [inline code/modules/admin/verbs/randomverbs.dm:715]: You are running get contents twice.

**Review by Archanial — CHANGES_REQUESTED** [review]

File: code/modules/admin/verbs/randomverbs.dm (hunk 1 above)
Comment (ike709) [reply code/modules/admin/verbs/randomverbs.dm:715]: Dupe of my comment.

**Comment by ike709** [comment]
Stale. Address my comment.

**Comment by PowerfulBacon** [comment]
Also printing the list directly won't work, it will just put /list into the log

**Pull request closed without merging**